docs/DesignDocs/RemoteAttestationCollaterals.md

```diff
@@ -0,0 +1,457 @@
+Remote Attestation Endorsements
+=====
+
+Attestation is the process of proving the authenticity of a
+Trusted Execution Environment (TEE) platform (HW + software).
+
+In the current API, OE SDK has functions to get the evidence and to verify
+the evidence.  This document describes additional changes to the API to
+support the concept of endorsements. Endorsements are additional information
+from a second source that the verifier can use for attestation verification.
+
+Terminology
+----------
+From Remote Attestation Procedures Architecture
+(https://www.ietf.org/id/draft-birkholz-rats-architecture-02.txt).
```

Comment:
Should note that that draft is a Work in Progress and subject to change.

Reply:
Added.